src/gitops/utils/gitops.test.ts

```
@@ -1,12 +1,30 @@
+  getApplicationArgoUrl,
   getFriendlyClusterName,
   getDuration,
   isApplicationRefreshing,
   getOperationType,
   getAppSetStatus,
 } from './gitops';
 
+describe('getApplicationArgoUrl', () => {
+  it('builds application URLs', () => {
+    expect(
+      getApplicationArgoUrl({ protocol: 'https', host: 'openshift-gitops-server-openshift-gitops' }, {
```

ESLint

[error] Replace `{·protocol:·'https',·host:·'openshift-gitops-server-openshift-gitops'·},` with `⏎········{·protocol:·'https',·host:·'openshift-gitops-server-openshift-gitops'·},⏎·······`

(prettier/prettier)